das_getter.py

```diff
@@ -0,0 +1,305 @@
+import httplib
+import os
+import json
+
+def cmsweb_get(url):
+    headers = {'Content-Type': 'application/json',
+               'Accept': 'application/json'}
+    cert = os.getenv('X509_USER_PROXY')
+    key = os.getenv('X509_USER_PROXY')
+    print('Using certificate and key from %s' % cert)
+    if cert is None:
+        print('No X509_USER_PROXY found! Run "voms-proxy-init -voms cms; export X509_USER_PROXY=$(voms-proxy-info --path)" and try again')
+        return {}
+
+    conn = httplib.HTTPSConnection('cmsweb.cern.ch', cert_file=cert, key_file=key)
+    conn.request("GET", url, headers=headers)
+    response = conn.getresponse()
+    status, data = response.status, response.read()
+    conn.close()
+    print('HTTP status: %s' % (status))
+    jsonfile = json.loads(data)
```

Comment:
This should be in `try` block as it catches when returned data is not a valid JSON